examples/resources/aci_tag/resource-all-attributes.tf

```diff
@@ -0,0 +1,12 @@
+
```

Comment:
Is there a case about removing the all-attributes one if they are both identical?

Reply:
changed